toolchain/install/install_filegroups.bzl

```
@@ -85,7 +90,7 @@ def make_install_filegroups(name, no_driver_name, pkg_name, install_dirs, prefix
             if not entry["is_driver"]:
                 no_driver_srcs.append(prefixed_path)
 
-            pkg_path = path + ".pkg"
+            pkg_path = entry.get("label") or path + ".pkg"
             pkg_srcs.append(pkg_path)
```

Name seems a bit misleading now:

```suggestion
            pkg_label = entry.get("label") or path + ".pkg"
            pkg_srcs.append(pkg_label)
```

I tihnk this will need more changes beyond the window I can suggest edits in, but I think they're all correct as labels.